docs/src/markdown-pages/loop-approval-checklist.md

```diff
@@ -0,0 +1,42 @@
+---
+slug: 'guides/loop-approval'
+title: 'Loop Approval Checklist'
+description: 'A checklist of security, design, and other requirements to get a loop approved for the library.'
+---
+
+## Olive Helps Loop Approval Requirements
+
+**Security & technical Requirements for Loop Authors**
+
+1. Ensure that all Loop information and metadata is complete and accurate. Please provide a Loop Description that includes the following:
+
+   - License Agreement (can be a link to an external resource)
+   - Privacy Policy (can be a link to an external resource)
+   - Declare what sensors your loop uses from the LDK
+   - Must declare how your Loop uses the user’s information and how your organization uses that information
```

Comment:
Should it be `Loop` or `loop`?

Reply:
I asked Robin, and she said that it should be a proper noun. So `Loop`.